src/flow-spec/IBlock.ts

```diff
@@ -319,16 +319,13 @@ export function wrapInExprSyntaxWhenAbsent(expr: string): string {
 }
 
 /**
- * Set a property on the contact contained in the flow context.
+ * Set properties on the contact contained in the flow context.
  */
 export function setContactProperty<BLOCK_CONFIG extends ISetContactPropertyBlockConfig>(
   block: IBlock<BLOCK_CONFIG>,
   context: IContext
 ): void {
-  const setContactProperty = block.config.set_contact_property
-  if (setContactProperty != null) {
-    setSingleContactProperty(setContactProperty, context)
-  }
+  block.config.set_contact_property?.forEach(setContactProperty => setSingleContactProperty(setContactProperty, context))
```

Comment:
@bulatgab : We will not apply the update in block.config.set_contact_property, we will only consider it in the SetContactProperty block. See confirmation from George: https://votomobile.slack.com/archives/CMQDVRDN3/p1657037683855219?thread_ts=1656948302.778729&cid=CMQDVRDN3

cc: @object-required 

- [ ] revert changes related to other blocks's block.config.set_contact_property

Reply:
After our call, a quick note:
Maybe we can consider to make conditional value assignment here.

Reply:
Last decision on this from the crew: we will consider all `block.config.set_contact_property` to be array, even for other blocks. So, I'm deleting this comment as it's not relevant anymore